misc/dictation.py

```diff
@@ -0,0 +1,150 @@
+from talon import Context, actions
+
+ctx = Context()
+dictate = actions.dictate
+
+# wordwordword ...
+def smash(m):
+  smash = ""
```

Comment:
Can you follow the Python rule that indentation-is-four-spaces?

Reply:
Yep yep, thanks for pointing that out. I added the spaces in my 2nd commit